tests/frontend/test_circuitspecs_chip2.py

```diff
@@ -368,28 +369,32 @@ def test_no_unitary(self, tol):
             ops.S2gate(SQ_AMPLITUDE, 0) | (q[3], q[7])
 
             # corresponds to an identity on modes [0, 1, 2, 3]
-            ops.MZgate(0, 0) | (q[0], q[1])
-            ops.MZgate(0, 0) | (q[2], q[3])
+            # This can be easily seen from below by noting that:
+            # MZ(pi, pi) = R(0) = I
+            # MZ(pi, 0) @ MZ(pi, 0) = I
+            # R(pi) @ MZ(pi, 0) = I
+            ops.MZgate(np.pi, 0) | (q[0], q[1])
+            ops.MZgate(np.pi, 0) | (q[2], q[3])
+            ops.MZgate(np.pi, np.pi) | (q[1], q[2])
+            ops.MZgate(np.pi, np.pi) | (q[0], q[1])
+            ops.MZgate(np.pi, 0) | (q[2], q[3])
```

Comment:
Note: the change here was required due to merging in #311. This altered the way the decomposition represents the identity unitary. Have added a comment to help explain why this is the identity.